src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/latest/test_dns_commands.py

```diff
@@ -150,11 +150,9 @@ def test_dns(self, resource_group):
 
         self.cmd('network dns record-set a remove-record -g {rg} --zone-name {zone} --record-set-name myrsa --ipv4-address 10.0.0.11')
 
-        self.cmd('network dns record-set a show -n myrsa -g {rg} --zone-name {zone}',
-                 checks=self.is_empty())
+        self.cmd('network dns record-set a show -n myrsa -g {rg} --zone-name {zone}', expect_failure=True)
```

Comment:
The change needed to this test implies there is a serious breaking change here.

Reply:
True